src/specs/01_basic/multi_tenancy.spec

```diff
@@ -0,0 +1,297 @@
+    | `private://Wallet1/_/contracts?owned.eoa=0x0&from.tm=JPM_K1`  |
+* Configure `"JPM Settlement"` in authorization server with:
+    | scope                                                         |
+    |---------------------------------------------------------------|
+    | `rpc://eth_*` |
+    | `private://JPM_ACC2/_/contracts?owned.eoa=0x0&from.tm=JPM_K2` |
+* Configure `"JPM Audit"` in authorization server with:
+    | scope                                                            |
+    |------------------------------------------------------------------|
+    | `rpc://eth_*`                                                    |
+    | `private://0x0/read/contracts?owned.eoa=JPM_ACC1&from.tm=JPM_K1` |
+    | `private://0x0/read/contracts?owned.eoa=JPM_ACC2&from.tm=JPM_K2` |
+* Configure `"GS Investment"` in authorization server with:
+    | scope                                                       |
+    |-------------------------------------------------------------|
+    | `rpc://eth_*`                                               |
+    | `private://GS_ACC1/_/contracts?owned.eoa=0x0&from.tm=GS_K1` |
+    | `private://Wallet2/_/contracts?owned.eoa=0x0&from.tm=GS_K1` |
+* Configure `"GS Settlement"` in authorization server with:
+    | scope                                                       |
+    |-------------------------------------------------------------|
+    | `rpc://eth_*`                                               |
+    | `private://GS_ACC2/_/contracts?owned.eoa=0x0&from.tm=GS_K2` |
+    | `private://Wallet2/_/contracts?owned.eoa=0x0&from.tm=GS_K2` |
+* Configure `"GS Research"` in authorization server with:
+    | scope                                                       |
+    |-------------------------------------------------------------|
+    | `rpc://eth_*`                                               |
+    | `private://GS_ACC3/_/contracts?owned.eoa=0x0&from.tm=GS_K3` |
+* Configure `"GS Audit"` in authorization server with:
+    | scope                                                          |
+    |----------------------------------------------------------------|
+    | `rpc://eth_*`                                                  |
+    | `private://0x0/read/contracts?owned.eoa=GS_ACC1&from.tm=GS_K1` |
+    | `private://0x0/read/contracts?owned.eoa=GS_ACC2&from.tm=GS_K2` |
+    | `private://0x0/read/contracts?owned.eoa=GS_ACC3&from.tm=GS_K3` |
+* Configure `"DB Investment"` in authorization server with:
+    | scope                                                       |
+    |-------------------------------------------------------------|
+    | `rpc://eth_*`                                               |
+    | `private://DB_ACC1/_/contracts?owned.eoa=0x0&from.tm=DB_K1` |
+    | `private://Wallet8/_/contracts?owned.eoa=0x0&from.tm=DB_K1` |
+
+## JPM using self-managed account keys can only deploy private contracts from the allocated TM keys
+
+tags: private, deploy, raw
```

Comment:
MINOR: Would it make sense to add a multitenancy tag?

Reply:
the scenario tag list inherits tags from the spec so we don't need to declare here

Reply:
Thanks